src/v/rpc/test/redpanda.other.crt

```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
@mmaslankaprv wrote a pythong script probably worth extending our cmake python w/ i think in the vtools directory to automatically generate certs, etc.

Reply:
the script is part of the vtools but the certs are generated inside the redpanda, it won't be possible to regenerate them as part of the open-source build
but it can be done in standalone script (similar to rpcgen.py)

Reply:
I didn't include the script and changes to cmake into the patch because at this point it can't be done optimally and we only have a fixed number of credentials in the tree. The problem is that `INPUT_FILES` in the rp_test macro doesn't introduce dependency so the custom command that invokes the script is attached to target `ALL`. Because of that credentials are regenerated on every build.